scripts/jupyterlite/extra_packages.json

```diff
@@ -1,4 +1,3 @@
 [
-  "https://cdn.holoviz.org/panel/wheels/panel-1.5.5-py3-none-any.whl",
-  "pyodide-http"
+  "https://cdn.holoviz.org/panel/wheels/panel-1.5.5-py3-none-any.whl"
```

Comment:
Remind me, do we automatically bump this somehow?

Reply:
No, we want a version that is recent enough so that it can solve it. The panel and bokeh install inside the jupyterlite will be the latest of our pyodide builds. 

 When trying to bump Panel to the latest, I hit the micropip problem.